mkt/submit/views.py

```diff
@@ -136,7 +137,7 @@ def manifest(request):
 
         # Call task outside of `atomic` to avoid it running before
         # the transaction is committed and not finding the app.
```

Comment:
Can you also remove this comment ? It's obsolete, we are no longer manually using `atomic` here, instead it's a `post_request_task` automatically delayed till the end of the request-response cycle (which is wrapped in a transaction with `ATOMIC_REQUESTS`)